packages/documentsearch/src/providers/notebooksearchprovider.ts

```diff
@@ -359,6 +371,13 @@ export class NotebookSearchProvider implements ISearchProvider<NotebookPanel> {
     return this._changed;
   }
 
+  /**
+   * Signal indicating that the active cell in the notebook has changed
+   */
+  get activeCellChanged(): ISignal<this, void> {
```

Comment:
I may be missing something, but it appears to me that this signal is not in use (neither emitted or listened to). Is this correct?

Reply:
Not in use indeed, and I don't think it's needed anymore. Will clean up :) thanks @krassowski

Reply:
Resolved by #10169. Thanks!